panic: runtime error: invalid memory address or nil pointer dereference

### Summary
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x8d272a]

goroutine 1 [running]:
github.com/gopasspw/gopass/internal/store/leaf.(*Store).idFiles(0x10?, {0xea9b18?, 0xc000738270?})
        work/gopass/gopass/internal/store/leaf/store.go:111 +0x2a
github.com/gopasspw/gopass/internal/store/leaf.(*Store).RecipientsTree(0x20?, {0xea9b18, 0xc000738270})
        work/gopass/gopass/internal/store/leaf/recipients.go:71 +0x37
github.com/gopasspw/gopass/internal/store/root.(*Store).RecipientsTree(0xc000400090, {0xea9b18, 0xc000738270}, 0x10?)
        work/gopass/gopass/internal/store/root/recipients.go:98 +0xcc
github.com/gopasspw/gopass/internal/action.(*Action).recipientsList(0xc000478000, {0xea9b18?, 0xc000404900?})
        work/gopass/gopass/internal/action/recipients.go:52 +0x65
github.com/gopasspw/gopass/internal/action.(*Action).RecipientsComplete(0x0?, 0x1?)
        work/gopass/gopass/internal/action/recipients.go:66 +0x2f
github.com/urfave/cli/v2.ShowCompletions(...)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:326
github.com/urfave/cli/v2.checkCompletions(0xc00072b280)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:475 +0xd3
github.com/urfave/cli/v2.(*Command).Run(0xc000723a20, 0xc00072b280, {0xc00073d600, 0x2, 0x2})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:158 +0x125
github.com/urfave/cli/v2.(*Command).Run(0xc000723600, 0xc00072b180, {0xc0007380c0, 0x3, 0x3})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*Command).Run(0xc0007274a0, 0xc00072a000, {0xc0000340a0, 0x4, 0x5})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*App).RunContext(0xc00047a000, {0xea9b18?, 0xc000404900}, {0xc0000340a0, 0x5, 0x5})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/app.go:332 +0x5b7
main.main()
        work/gopass/gopass/main.go:90 +0x4c5
panic: runtime error: invalid memory address or nil pointer dereference


### Steps To Reproduce
- gopass recipients rm  <TAB>

### Expected behavior
List mounts, and users

### Environment
<!--
Please complete the following information (see note below)
-->

- OS: [e.g. Mac OS X High Sierra, Ubuntu 18.04, Windows 10, ...] Ubuntu 22.04
- OS version: [uname -a] Linux yyyyyyyy  SMP PREEMPT_DYNAMIC Thu Jan 18 02:29:42 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
- gopass Version: [gopass version] gopass 1.15.11 go1.21.4 linux amd64
- Installation method: [e.g. from source, brew, gopass repo] 
```
dpkg -l | grep gopass
ii  gopass                                        1.15.11                                      amd64        gopass password manager - full featured CLI replacement for pass, designed for teams.
``` 

<!--
**PLEASE NOTE**

There is a package named gopass in the official Debian repository.
This package is not related to this project in any way. If you
installed gopass from the Debian archives report any bugs in
the Debian BTS.
-->

### Additional context
<!--
Add any other context about the problem here.
-->

```
GOPASS_DEBUG_LOG=/tmp/gopass.log gopass recipients rm panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x8d272a]

goroutine 1 [running]:
github.com/gopasspw/gopass/internal/store/leaf.(*Store).idFiles(0x10?, {0xea9b18?, 0xc00010f200?})
        work/gopass/gopass/internal/store/leaf/store.go:111 +0x2a
github.com/gopasspw/gopass/internal/store/leaf.(*Store).RecipientsTree(0x20?, {0xea9b18, 0xc00010f200})
        work/gopass/gopass/internal/store/leaf/recipients.go:71 +0x37
github.com/gopasspw/gopass/internal/store/root.(*Store).RecipientsTree(0xc000120210, {0xea9b18, 0xc00010f200}, 0x10?)
        work/gopass/gopass/internal/store/root/recipients.go:98 +0xcc
github.com/gopasspw/gopass/internal/action.(*Action).recipientsList(0xc000544000, {0xea9b18?, 0xc0004f09c0?})
        work/gopass/gopass/internal/action/recipients.go:52 +0x65
github.com/gopasspw/gopass/internal/action.(*Action).RecipientsComplete(0x0?, 0x0?)
        work/gopass/gopass/internal/action/recipients.go:66 +0x2f
github.com/urfave/cli/v2.ShowCompletions(...)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:326
github.com/urfave/cli/v2.checkCompletions(0xc000296900)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:475 +0xd3
github.com/urfave/cli/v2.(*Command).Run(0xc00054fa20, 0xc000296900, {0xc00004d870, 0x1, 0x1})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:158 +0x125
github.com/urfave/cli/v2.(*Command).Run(0xc00054f600, 0xc000296800, {0xc000127b00, 0x2, 0x2})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*Command).Run(0xc0005514a0, 0xc000035580, {0xc000140040, 0x3, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*App).RunContext(0xc000546000, {0xea9b18?, 0xc0004f09c0}, {0xc000140040, 0x4, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/app.go:332 +0x5b7
main.main()
        work/gopass/gopass/main.go:90 +0x4c5
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x8d272a]

goroutine 1 [running]:
github.com/gopasspw/gopass/internal/store/leaf.(*Store).idFiles(0x10?, {0xea9b18?, 0xc000524210?})
        work/gopass/gopass/internal/store/leaf/store.go:111 +0x2a
github.com/gopasspw/gopass/internal/store/leaf.(*Store).RecipientsTree(0x20?, {0xea9b18, 0xc000524210})
        work/gopass/gopass/internal/store/leaf/recipients.go:71 +0x37
github.com/gopasspw/gopass/internal/store/root.(*Store).RecipientsTree(0xc0003b0090, {0xea9b18, 0xc000524210}, 0x10?)
        work/gopass/gopass/internal/store/root/recipients.go:98 +0xcc
github.com/gopasspw/gopass/internal/action.(*Action).recipientsList(0xc0004e0000, {0xea9b18?, 0xc0003ee900?})
        work/gopass/gopass/internal/action/recipients.go:52 +0x65
github.com/gopasspw/gopass/internal/action.(*Action).RecipientsComplete(0x0?, 0x0?)
        work/gopass/gopass/internal/action/recipients.go:66 +0x2f
github.com/urfave/cli/v2.ShowCompletions(...)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:326
github.com/urfave/cli/v2.checkCompletions(0xc0004ff300)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:475 +0xd3
github.com/urfave/cli/v2.(*Command).Run(0xc0004eda20, 0xc0004ff300, {0xc0004efc20, 0x1, 0x1})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:158 +0x125
github.com/urfave/cli/v2.(*Command).Run(0xc0004ed600, 0xc0004ff200, {0xc0005231e0, 0x2, 0x2})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*Command).Run(0xc0004f94a0, 0xc0004fe080, {0xc0000340c0, 0x3, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*App).RunContext(0xc0004e2000, {0xea9b18?, 0xc0003ee900}, {0xc0000340c0, 0x4, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/app.go:332 +0x5b7
main.main()
        work/gopass/gopass/main.go:90 +0x4c5
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x8d272a]

goroutine 1 [running]:
github.com/gopasspw/gopass/internal/store/leaf.(*Store).idFiles(0x10?, {0xea9b18?, 0xc000766210?})
        work/gopass/gopass/internal/store/leaf/store.go:111 +0x2a
github.com/gopasspw/gopass/internal/store/leaf.(*Store).RecipientsTree(0x20?, {0xea9b18, 0xc000766210})
        work/gopass/gopass/internal/store/leaf/recipients.go:71 +0x37
github.com/gopasspw/gopass/internal/store/root.(*Store).RecipientsTree(0xc000700090, {0xea9b18, 0xc000766210}, 0x10?)
        work/gopass/gopass/internal/store/root/recipients.go:98 +0xcc
github.com/gopasspw/gopass/internal/action.(*Action).recipientsList(0xc00073e000, {0xea9b18?, 0xc000704900?})
        work/gopass/gopass/internal/action/recipients.go:52 +0x65
github.com/gopasspw/gopass/internal/action.(*Action).RecipientsComplete(0x0?, 0x0?)
        work/gopass/gopass/internal/action/recipients.go:66 +0x2f
github.com/urfave/cli/v2.ShowCompletions(...)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:326
github.com/urfave/cli/v2.checkCompletions(0xc000755300)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:475 +0xd3
github.com/urfave/cli/v2.(*Command).Run(0xc00074ba20, 0xc000755300, {0xc00074dc40, 0x1, 0x1})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:158 +0x125
github.com/urfave/cli/v2.(*Command).Run(0xc00074b600, 0xc000755200, {0xc0007651e0, 0x2, 0x2})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*Command).Run(0xc00074f4a0, 0xc000754080, {0xc0000340c0, 0x3, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*App).RunContext(0xc000740000, {0xea9b18?, 0xc000704900}, {0xc0000340c0, 0x4, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/app.go:332 +0x5b7
main.main()
        work/gopass/gopass/main.go:90 +0x4c5
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x8d272a]

goroutine 1 [running]:
github.com/gopasspw/gopass/internal/store/leaf.(*Store).idFiles(0x10?, {0xea9b18?, 0xc0005e6210?})
        work/gopass/gopass/internal/store/leaf/store.go:111 +0x2a
github.com/gopasspw/gopass/internal/store/leaf.(*Store).RecipientsTree(0x20?, {0xea9b18, 0xc0005e6210})
        work/gopass/gopass/internal/store/leaf/recipients.go:71 +0x37
github.com/gopasspw/gopass/internal/store/root.(*Store).RecipientsTree(0xc0003e8090, {0xea9b18, 0xc0005e6210}, 0x10?)
        work/gopass/gopass/internal/store/root/recipients.go:98 +0xcc
github.com/gopasspw/gopass/internal/action.(*Action).recipientsList(0xc0005c0000, {0xea9b18?, 0xc0003ec900?})
        work/gopass/gopass/internal/action/recipients.go:52 +0x65
github.com/gopasspw/gopass/internal/action.(*Action).RecipientsComplete(0x0?, 0x0?)
        work/gopass/gopass/internal/action/recipients.go:66 +0x2f
github.com/urfave/cli/v2.ShowCompletions(...)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:326
github.com/urfave/cli/v2.checkCompletions(0xc0005d5300)
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/help.go:475 +0xd3
github.com/urfave/cli/v2.(*Command).Run(0xc0005cda20, 0xc0005d5300, {0xc0005cfc30, 0x1, 0x1})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:158 +0x125
github.com/urfave/cli/v2.(*Command).Run(0xc0005cd600, 0xc0005d5200, {0xc0005e51e0, 0x2, 0x2})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*Command).Run(0xc0005d14a0, 0xc0005d4080, {0xc0000340c0, 0x3, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/command.go:267 +0xbe5
github.com/urfave/cli/v2.(*App).RunContext(0xc0005c2000, {0xea9b18?, 0xc0003ec900}, {0xc0000340c0, 0x4, 0x4})
        go/pkg/mod/github.com/urfave/cli/v2@v2.25.7/app.go:332 +0x5b7
main.main()
        work/gopass/gopass/main.go:90 +0x4c5
^C
```

no created logfile

--- a/internal/action/recipients.go
+++ b/internal/action/recipients.go
@@ -63,6 +63,12 @@ func (s *Action) recipientsList(ctx context.Context) []string {
 // completion.
 func (s *Action) RecipientsComplete(c *cli.Context) {
 	ctx := ctxutil.WithGlobalFlags(c)
+	if err := s.IsInitialized(c); err != nil {
+		debug.Log("IsInitialized returned error: %s", err)
+
+		return
+	}
+
 	for _, v := range s.recipientsList(ctx) {
 		fmt.Fprintln(stdout, v)
 	}

--- a/internal/store/leaf/store.go
+++ b/internal/store/leaf/store.go
@@ -108,7 +108,7 @@ func (s *Store) idFile(ctx context.Context, name string) string {
 
 // idFiles returns the path to all id files in this store.
 func (s *Store) idFiles(ctx context.Context) []string {
-	if s.crypto == nil {
+	if s == nil || s.crypto == nil {
 		return nil
 	}
 

